paddle/fluid/operators/split_op.cc

```diff
@@ -73,18 +73,17 @@ class SplitOp : public framework::OperatorWithKernel {
  protected:
   framework::OpKernelType GetExpectedKernelType(
       const framework::ExecutionContext &ctx) const override {
-    return framework::OpKernelType(ctx.Input<framework::LoDTensor>("X")->type(),
-                                   ctx.device_context());
-  }
-
-  framework::OpKernelType GetKernelTypeForVar(
```

Comment:
removed this function just for testing purposes, it needs to be restored and changed before this PR will be merged